docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -0,0 +1,121 @@
+|------|-------------|-----------------|--------------|
+| `javaParameters` | Generate metadata for Java 1.8 reflection on method parameters |  | false |
+| `jdkHome` | Include a custom JDK from the specified location into the classpath instead of the default JAVA_HOME. Direct setting is not possible, use [other ways to set this option](gradle-configure.md#set-custom-jdk-home).  |  |  |
+| `jvmTarget` | Target version of the generated JVM bytecode | "1.8", "9", "10", ..., "18" | "%defaultJvmTargetVersion%" |
+| `noJdk` | Don't automatically include the Java runtime into the classpath |  | false |
+| `useOldBackend` | Use the [old JVM backend](whatsnew15.md#stable-jvm-ir-backend) |  | false |
+
+### Attributes common to JVM, JS, and JS DCE
+
+| Name | Description | Possible values |Default value |
+|------|-------------|-----------------|--------------|
+| `allWarningsAsErrors` | Report an error if there are any warnings |  | false |
+| `suppressWarnings` | Don't generate warnings |  | false |
+| `verbose` | Enable verbose logging output. Works only when the [Gradle debug log level enabled](https://docs.gradle.org/current/userguide/logging.html) |  | false |
+| `freeCompilerArgs` | A list of additional compiler arguments |  | [] |
+
+### Attributes common to JVM and JS
+
+| Name | Description | Possible values |Default value |
+|------|-------------|-----------------|--------------|
+| `apiVersion` | Restrict the use of declarations to those from the specified version of bundled libraries | "1.3" (DEPRECATED), "1.4" (DEPRECATED), "1.5", "1.6", "1.7" |  |
+| `languageVersion` | Provide source compatibility with the specified version of Kotlin | "1.4" (DEPRECATED), "1.5", "1.6", "1.7" |  |
+
+### Attributes specific to JS
+
+| Name | Description | Possible values |Default value |
+|------|-------------|-----------------|--------------|
+| `friendModulesDisabled` | Disable internal declaration export |  | false |
+| `main` | Define whether the `main` function should be called upon execution | "call", "noCall" | "call" |
+| `metaInfo` | Generate .meta.js and .kjsm files with metadata. Use to create a library |  | true |
+| `moduleKind` | The kind of JS module generated by the compiler | "umd", "commonjs", "amd", "plain"  | "umd" |
+| `outputFile` | Destination *.js file for the compilation result |  | "\<buildDir>/js/packages/\<project.name>/kotlin/\<project.name>.js" |
+| `sourceMap` | Generate source map |  | true |
+| `sourceMapEmbedSources` | Embed source files into the source map | "never", "always", "inlining" |  |
+| `sourceMapPrefix` | Add the specified prefix to paths in the source map |  |  |
+| `target` | Generate JS files for specific ECMA version | "v5" | "v5" |
+| `typedArrays` | Translate primitive arrays to JS typed arrays |  | true |
+
+## What's next?
+
+Learn more about:
+* [Incremental compilation, caches support, build reports, and the Kotlin daemon](gradle-compilation-and-caches.md)
```

Comment:
```suggestion
* [Incremental compilation, cache support, build reports, and the Kotlin daemon](gradle-compilation-and-caches.md)
```

Reply:
There are two caches: the build cache and the configuration cache. Maybe "support for caches" would be better?

Reply:
Ah OK. Then it's best to leave it as it is.